VulkanHppGenerator.cpp

```diff
@@ -14347,7 +14335,8 @@ void VulkanHppGenerator::readTypeDefine( tinyxml2::XMLElement const * element, s
 
     checkForError( m_types.insert( { name, TypeData{ TypeCategory::Define, {}, line } } ).second, line, "define <" + name + "> already specified" );
     assert( m_defines.find( name ) == m_defines.end() );
-    m_defines[name] = { deprecated, require, line, deprecatedReason, possibleCallee, params, possibleDefinition };
+    m_defines[name]    = { deprecated, require, line, deprecatedReason, possibleCallee, params, possibleDefinition };
+    m_definesPartition = partitionDefines( m_defines );
```

Comment:
Good idea to partition the defines at just one place in code.
But I think, it should be done just once, not each time after adding a define.
So, this call should probably moved to near the end of the VulkanHppGenerator constructor.